Assets/MixedRealityToolkit.Providers/WindowsMixedReality/WindowsMixedRealityEyeGazeDataProvider.cs

```diff
@@ -51,14 +51,27 @@ public class WindowsMixedRealityEyeGazeDataProvider : BaseInputDeviceManager, IM
         private Ray saccade_initialGazePoint;
         private List<Ray> saccade_newGazeCluster = new List<Ray>();
 
-#if WINDOWS_UWP
-        private static bool askedForETAccessAlready = false; // To make sure that this is only triggered once.
-#endif
-
         public event Action OnSaccade;
         public event Action OnSaccadeX;
         public event Action OnSaccadeY;
 
+#if WINDOWS_UWP
+
+        private static bool askedForETAccessAlready = false; // To make sure that this is only triggered once.
+
+        /// <inheritdoc/ >
+        public override bool CheckCapability(MixedRealityInputCapabilities capability)
+        {
+            if (WindowsApiChecker.UniversalApiContractV8_IsAvailable)
+            {
+                return (EyesPose.IsSupported() && (capability == MixedRealityInputCapabilities.EyeTracking));
```

Comment:
Nit: Consider doing the cheap check first (i.e. capability == EyeTracking) followed by the possibly not cheap check.

Ideally you'd want to short circuit in the order of having faster things check first.

In reality I doubt this has any real impact here, but it's generally a good practice/habit to get into.